docs/reference/commandline/events.md

```diff
@@ -51,6 +51,7 @@ The currently supported filters are:
 * container
 * event
 * image
+* labels
```

Comment:
`key=value` is supported, it works the same way as filters on images or containers